### DISCUSSION TITLE
Switch from G7 to G6 - revert to previous settings

### DISCUSSION THREAD

**Discussion by rudiru** [NightscoutFoundation/xDrip#3335] [Q&A]
Hi,
i have configured xdrip with these settings ( https://navid200.github.io/xDrip/docs/Dexcom/G7.html ) to work with G7 sensors, and i would like to go back permanently to G6.

Currently I enter the Transmitter ID and am not able to set a sensor Code in xdrip (because it is not needed and therefore skipped).

Which configurations are needed to be able to start a G6 Sensor after adding a new G6 Transmitter?

BR

ru

**Comment by Navid200** [comment]
This walks you through setting up xDrip for using G6:
https://navid200.github.io/xDrip/docs/Starting-G6.html

**Comment by rudiru** [comment]
workflow is not working for scenario of changing back from G7 to G6.

There is no prompt to enter the sensor ID after hitting "start sensor"

**Comment by rudiru** [comment]
had to reinstall and lost all the settings

**Comment by Navid200** [comment]
I am sorry about your experience.
Nothing changes in xDrip by using G7 that makes it not work as it did before with G6.

**Comment by rudiru** [comment]
the change is that there is no way to enter a sensor id anymore after applying the G7 settings. there was no way to enter that anymore, which is necessary for G6

**Reply by Navid200** [reply]
I just tested this.

I admit that we do have a problem with handling the start time of the sensor with G7.
When I switched back to G6, xDrip assumed that there was a session in progress and did not give me the option to start.  In order to deal with that, I waited for a 5-minute connectivity cycle to complete.  Then, right away, I stopped sensor.  But, before this command could be sent to the transmitter, I cleared the queue: https://navid200.github.io/xDrip/docs/Clear-queue.html

After that, I was able to start sensor and xDrip asked for the 4-digit calibration code.
